analytics/policy/docs/sw_design_doc.md

```diff
@@ -0,0 +1,76 @@
+# Software Design Doc
+
+Status: Approved (2024-01-27)
+
+Author: Nam Mai
+
+Reviewer(s): Yang Su
+
```

Comment:
i am guessing this is duplicated from the wiki. try not duplicate information because it will cause issue when you update information. try to keep a single source of truth. if you want to refer to the data, simply put the link to the source of truth.

Reply:
> i am guessing this is duplicated from the wiki. try not duplicate information because it will cause issue when you update information. try to keep a single source of truth. if you want to refer to the data, simply put the link to the source of truth.

I put the doc here since it's better to version control here. But wiki might have better visibility
Should I keep the doc here or wiki?

Reply:
imagine if you were a new dev here where would u prefer to find the info. and what happens when there are conflicting info in two different places. make decision from this thought process

Reply:
> imagine if you were a new dev here where would u prefer to find the info. and what happens when there are conflicting info in two different places. make decision from this thought process

Will remove this one and keep wiki for ease of use at the moment